processor/src/test/java/org/mapstruct/ap/testutil/runner/AnnotationProcessorTestRunner.java

```diff
@@ -57,6 +70,12 @@ public AnnotationProcessorTestRunner(Class<?> klass) throws Exception {
         if (singleCompiler != null) {
             return Arrays.<Runner> asList( new InnerAnnotationProcessorRunner( klass, singleCompiler.value() ) );
         }
+        else if ( IS_AT_LEAST_JAVA_9 ) {
+            // Current tycho-compiler-jdt (0.26.0) is not compatible with Java 11
+            // Updating to latest version 1.3.0 fails some tests
+            // Once https://github.com/mapstruct/mapstruct/pull/1587 is resolved we can remove this line
+            return Arrays.asList( new InnerAnnotationProcessorRunner( klass, Compiler.JDK11 ) );
```

Comment:
"is at least Java 9" and then providing JDK 11 looks a bit strange for me... if it is Java 9, 10, 12, ... then JDK11 compiler is correct as well?!

Reply:
Not sure from which Java the change was done, but we are testing with Java 11 so I think it is fine